### DISCUSSION TITLE
The number of vertices in the original FBX file has changed after being parsed by the SDK.

### DISCUSSION THREAD

**Discussion by sysuNie** [assimp/assimp#4499] [Q&A]
The number of vertices in the original FBX file has changed after being parsed by the SDK. Is it necessary to modify the flags in the FBX model read in the above way to solve the problem of the number of vertices changing?
Assimp::Importer m_importer;
m_assImpScene = m_importer.ReadFile(fileName,
                aiProcess_Triangulate //Triangulates all faces of all meshes
                | aiProcess_GenNormals); //Generate normals for meshes

**Comment by ssw-python** [comment]
I have met the same problem, it seems like the importer will copy the vertices by the face index. But i do not know how can i respect the origin topology from the input file.
